animepahe-dl.sh

```diff
@@ -32,6 +33,10 @@ set_var() {
     _FZF="$(command -v fzf)" || command_not_found "fzf"
     _NODE="$(command -v node)" || command_not_found "node"
     _FFMPEG="$(command -v ffmpeg)" || command_not_found "ffmpeg"
+    [[ ${_PARALLEL_RUN:-} = "true" ]] && {
+       _XXD="$(command -v xxd)" || command_not_found "xxd"
+       _OPENSSL="$(command -v openssl)" || command_not_found "openssl"
+    }
```

Comment:
Since it's already 2 lines, I suggest to use the whole if structure:
```
if ...; then
  ...
fi
```

Reply:
Resolved Snippet:
```
    if [[ ! -z ${_PARALLEL_JOBS:-} ]]; then
       _XXD="$(command -v xxd)" || command_not_found "xxd"
       _OPENSSL="$(command -v openssl)" || command_not_found "openssl"
    fi
    
```